### PR TITLE
[RFR] Test e2e field reorder

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

File: cypress/integration/model.spec.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        homePage.goToAdminDashboard();
+        datasetImportPage.importDataset('dataset/simple.csv');
+        datasetImportPage.importModel('model/concat.json');
+        datasetImportPage.publish();
+        datasetImportPage.goToModel();
+    });
+
+    describe('should display characteristic fields', () => {
+        it('should display list of characteristics and allow to reorder them', () => {
+            cy.visit('/');
```
Comment (Kmaschta) [inline cypress/integration/model.spec.js]: Can you move all these useful fonctions into a proper support please?

**Review by Kmaschta — CHANGES_REQUESTED** [review]

**Review by Kmaschta — APPROVED** [review]

**Review by Kmaschta — CHANGES_REQUESTED** [review]

**Comment by ThieryMichel** [comment]
I forced the langage to be english for test

**Review by Kmaschta — CHANGES_REQUESTED** [review]
English seems to not be enforced on my desktop.
Are you sure that the enforcement is working?

**Comment by ThieryMichel** [comment]
fixed

File: cypress/integration/model.spec.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        datasetImportPage.importModel('model/concat.json');
+        datasetImportPage.publish();
+        datasetImportPage.goToPublishedResources();
+    });
+
+    it('should display list of characteristics and allow to reorder them', () => {
+        homePage.checkCharacteristicsOrder(['Title', 'Description']);
+        homePage.goToAdminDashboard();
+        datasetImportPage.goToModel();
+        cy.get('.ontology-table-dataset h4').contains('Dataset');
```
Comment (Kmaschta) [inline cypress/integration/model.spec.js]: This is not a test. You have to add a `.should` function to test something.

File: cypress/integration/model.spec.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    });
+
+    it('should display list of resource field and allow to reorder them', () => {
+        homePage.goToGraphPage();
+        graphPage.checkColumnHeaders(['#', 'Column 1', 'Column 2']);
+
+        homePage.goToAdminDashboard();
+        datasetImportPage.goToModel();
+
+        cy.get('.ontology-table-dataset h4').contains('Dataset');
```
Comment (Kmaschta) [inline cypress/integration/model.spec.js]: This is not a test. You have to add a `.should` function to test something.

File: cypress/integration/model.spec.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        modelPage.changeFilter('List of fields dedicated to each resource');
+        cy.get('.ontology-table-dataset').should('not.exist');
+        cy.get('.ontology-table-document h4').contains('Document');
```
Comment (Kmaschta) [inline cypress/integration/model.spec.js]: This is not a test. You have to add a `.should` function to test something.

File: cypress/support/graphPage.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+export const checkColumnHeaders = headers => {
+    headers.forEach((col, index) => {
+        cy.get(`.dataset table thead th:nth-child(${index + 1})`).contains(col);
```
Comment (Kmaschta) [inline cypress/support/graphPage.js]: This is not a test. You have to add a `.should` function to test something.

File: cypress/support/homePage.js [hunk 6]
```diff
@@ -1,9 +1,21 @@
...
+    cy
+        .get('nav a')
+        .contains('Resources')
+        .click();
     cy.location('pathname').should('equal', '/graph');
 };
+
+export const checkCharacteristicsOrder = names => {
+    names.forEach((name, index) =>
+        cy.get(`.property:nth-child(${index + 1})`).contains(name),
```
Comment (Kmaschta) [inline cypress/support/homePage.js]: This is not a test. You have to add a `.should` function to test something.

File: cypress/support/modelPage.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+};
+
+export const checkFieldOrder = type => names => {
+    cy
+        .get(`.ontology-table-${type} table tbody tr`)
+        .should('have.length', names.length);
+    names.forEach((name, index) => {
+        cy
+            .get(
+                `.ontology-table-${type} table tbody tr:nth-child(${index +
```
Comment (Kmaschta) [inline cypress/support/modelPage.js]: You can do better with this formatting..

**Review by Kmaschta — CHANGES_REQUESTED** [review]

**Comment by Kmaschta** [comment]
La PR de Jouvence.

**Review by Kmaschta — APPROVED** [review]

**Pull request merged**